mlptrain/sampling/plumed.py

```diff
@@ -1033,6 +1033,59 @@ def __init__(self, name: str, atom_groups: Sequence = None):
         )
 
 
+class PlumedCNCV(_PlumedCV):
+    def __init__(
+        self,
+        name: str,
+        ref: float,
+        n: int,
+        m: int,
+        atom_groups: Sequence = None,
+    ):
+        """
+        PLUMED collective variable as a coordination number (CN) between two atoms or groups of atoms
+
+        e.g. [(0, 2)] gives ζ =[1-(r_02/r_ref)^6]/ [1-(r_02/r_ref)^12]
+
+        which corresponds to the CN between atoms 0 and 2. To ensure that the CN has continuous derivatives,  we use a rational switching function consistently with PLUMED.
```

Comment:
We can split the comment to two lines. (Not sure why Ruff does not do it)
```suggestion
        which corresponds to the CN between atoms 0 and 2. To ensure that the CN has continuous derivatives,  
        we use a rational switching function consistent with PLUMED.
```